Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -54,6 +62,14 @@ final class SettingsAccountViewController: UIViewController {
       .observeValues { [weak self] (viewController) in
         self?.navigationController?.pushViewController(viewController, animated: true)
     }
+
+    self.viewModel.outputs.updateCurrency
+      .observeForUI()
+      .observeValues { selectedCurrency in
+        NotificationCenter.default.post(name: Notification.Name.ksr_updatedCurrencyCellDetailText,
```

Comment:
Here instead of posting a notification you could call `self?.dataSource.configureCurrencyRow(currency: selectedCurrency)` and either reload the table view or just the currency cell.